bundles/AdminBundle/Controller/Admin/DataObject/DataObjectController.php

```diff
@@ -1784,8 +1790,14 @@ public function gridProxyAction(
 
             $objects = [];
             foreach ($list->getObjects() as $object) {
-                $o = DataObject\Service::gridObjectData($object, $allParams['fields'] ?? null, $requestedLanguage,
-                    ['csvMode' => $csvMode]);
+                if($csvMode){
+                    $helperDefinitions = DataObject\Service::getHelperDefinitions();
+                    $o = DataObject\Service::getCsvDataForObject($object,$requestedLanguage,$request->get('fields'),DataObject\Service::getHelperDefinitions(),$localeService,false,$allParams['context']);
```

Comment:
`$helperDefinitions` is unused because its value is inlined in the argument.
```suggestion
                    $o = DataObject\Service::getCsvDataForObject($object,$requestedLanguage,$request->get('fields'),DataObject\Service::getHelperDefinitions(),$localeService,false,$allParams['context']);
```